coalib/processes/Processing.py

```diff
@@ -38,7 +39,8 @@
            ShowPatchAction,
            IgnoreResultAction,
            ShowAppliedPatchesAction,
-           GeneratePatchesAction]
+           GeneratePatchesAction,
+           AddNewLineAction]
```

Comment:
All the new actions created must be added here to support auto-applying of those actions.